ui/src/components/common/CopyToClipboardMultiline.scss

```diff
@@ -1,19 +1,27 @@
-.copy-to-clipboard-multiline {
-  display: block;
-  overflow-x: auto
-}
+pre {
```

Comment:
IMO we should keep `.copy-to-clipboard-multiline` (i.e. `.copy-to-clipboard-multiline pre {...}`) here as well, otherwise this styling would be applied to every `pre`.

Reply:
ok